core/frontend/src/components/autopilot/AutopilotManagerUpdater.vue

```diff
@@ -39,7 +39,7 @@ export default Vue.extend({
         .catch((error) => {
           autopilot.setAvailableEndpoints([])
           if (error === backend_offline_error) { return }
-          const message = `Could not fetch available MAVLink endpoints: ${error.message}`
+          const message = error.response.data.detail ? error.response.data.detail : error.message
```

Comment:
Forgot the coalescing operator existed. Will use.

Regarding having a single way, it is like that, but we never have a guarantee that the error we receive is indeed from our backend and so that it will respect our standard. Imagine a network interruption occurs. The received error will not have a body, so we should deal with it.